debian/control.in

```diff
@@ -43,18 +43,18 @@ Package: machinekit
 Breaks: linuxcnc
 Replaces: linuxcnc
 Architecture: any
-Depends: ${shlibs:Depends}, machinekit-rt-threads, @TCL_TK_DEPS@,
-    @DEPS@ bwidget (>= 1.7), libtk-img (>=1.13),
+Depends: ${shlibs:Depends}, machinekit-rt-threads, tcl8.6, tk8.6,
+    bwidget (>= 1.7), libtk-img (>=1.13),
     ${python:Depends}, ${misc:Depends},
     python-tk, python-imaging, python-imaging-tk,
     python-gnome2, python-glade2,
     python-numpy, python-gtksourceview2,
     python-vte, python-xlib, python-gtkglext1, python-configobj,
-    python-zmq, python-protobuf (>= 2.4.1), python-gst0.10,
+    python-zmq, python-protobuf (>= 2.4.1),
     python-avahi, python-simplejson, python-pyftpdlib,
-    python-pydot, xdot, libzmq3 (>= 4.0.4+dfsg-1~jessie), libczmq4 (>= 4.0.2-10~jessie),
+    python-pydot, xdot, libzmq3 (>= 4.0.4), libczmq4 (>= 4.0.2),
     tclreadline, bc, procps, psmisc,
-    gstreamer0.10-plugins-base
+    python-gst1.0, gstreamer1.0-plugins-base
```

Comment:
I wasn't aware that gstreamer1.o had been backported for Jessie, that makes life easier

Reply:
The package is actually called `python-gst-1.0`
https://packages.debian.org/jessie/python/python-gst-1.0
This fails installation

Then they quite happily call the plugins package `gstreamer1.0-plugins-base`, without the dash before the `1`.
Not much consistency there 😆